doc/ref_cert/RC2/chapters/chapter04.md

```diff
@@ -3,3 +3,38 @@
 # 4. CNF Test Cases and Requirements Traceability
 
 ## Table of Contents
+* [4.1 Introduction](#4.1)
+* [4.2 Selection Criteria](#4.2)
+* [4.3 Traceability Matrix](#4.3)
+  * [4.3.1 Test Case Traceability to CNF Interoperability Requirements](#4.3.1)
+  
+<a name="4.1"></a>
+## 4.1 Introduction
+The scope of this chapter is to identify and list down test cases based on requirements defined in [Reference Architecture-1 (RA-2)](../../../ref_arch/kubernetes/README.md). This will serve as traceability between test cases and requirements for CNF interoperability.
```

Comment:
```suggestion
The scope of this chapter is to identify and list down test cases based on requirements defined in [Reference Architecture-2 (RA-2)](../../../ref_arch/kubernetes/README.md). This will serve as traceability between test cases and requirements for CNF interoperability.
```